### DISCUSSION TITLE
How to delete all entry history at once?

### DISCUSSION THREAD

**Discussion by smed79** [keepassxreboot/keepassxc#10361] [Q&A] [answered]
Hey,

Is it possible to delete all entry histories for a specific group or the full database at once?

:broom:

**Comment by phoerious** [comment, marked as answer]
You can set your history size to 0 in the database settings, save the database, and then set it back to the previous value.

**Reply by smed79** [reply]
This workaround did the trick.

Thank you for the tips.
